database/database.go

```diff
@@ -1,62 +1,39 @@
 package database
 
 import (
-	"database/sql"
 	"fmt"
-	_ "github.com/mattn/go-sqlite3"
-	"github.com/thirdknife/scoutingapp/database/player"
-	"os"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
 )
 
-type Database struct {
-	db *sql.DB
-
-	PlayerTable *player.Table
-}
-
-// OpenOrCreate attempts to open the provided file, but if it doesn't exist it creates
-// one at the provided location. If no location is provided it creates an in-memory database.
-func OpenOrCreate(path string) (*Database, error) {
+// Load opens the database at the given path. If `path == ""` then a new in-memory database is returned.
+func Load(path string) (*gorm.DB, error) {
+	// Default to an in-memory database.
+	// https://gorm.io/docs/connecting_to_the_database.html#SQLite
 	if path == "" {
-		return New("")
-	}
-	if _, err := os.Stat(path); os.IsNotExist(err) {
-		return New(path)
-	}
-	return fromFile(path)
-}
-
-// New creates a new database at the given path. It fails if there is already a file at that path.
-func New(path string) (*Database, error) {
-	_, err := os.Stat(path)
-	if os.IsNotExist(err) {
-		// The file not existing is good. This method creates it.
-	} else {
-		return nil, fmt.Errorf("database file %q already exists", path)
+		path = "file::memory:?cache=shared"
```

Comment:
I guess this can be swapped with a database connection if postgres is needed?

Reply:
Yeah - "GORM officially supports the databases MySQL, PostgreSQL, SQLite, SQL Server, and TiDB"

https://gorm.io/docs/connecting_to_the_database.html#PostgreSQL